isort/info.yaml

```diff
@@ -0,0 +1,47 @@
+---
+enabled: true
+name: isort
+version: 4.3.21
+include:
+  - "**/*.py"
+interpreters:
+  - python
+documentation:
+  - https://github.com/timothycrosley/isort/
+metadata:
+  languages:
+    - Python
+  tests:
+    - contents: |
+        from my_lib import Object
+
+        import os
+
+        from my_lib import Object3
+
+        from my_lib import Object2
+
+        import sys
+
+        from third_party import lib15, lib1, lib2, lib3, lib4, lib5, lib6, lib7, lib8, lib9, lib10, lib11, lib12, lib13, lib14
+
+        import sys
+
+        from __future__ import absolute_import
+
+        from third_party import lib3
+
+        print("Hey")
+        print("yo")
+      restyled: |
+        from __future__ import absolute_import
+
+        import os
+        import sys
+
+        from my_lib import Object, Object2, Object3
+        from third_party import (lib1, lib2, lib3, lib4, lib5, lib6, lib7, lib8, lib9,
+                                 lib10, lib11, lib12, lib13, lib14, lib15)
+
+        print("Hey")
+        print("yo")
```

Comment:
This file is missing the final newline. Aside from the [general convention of always ending files with a newline](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)(which I prefer to follow), this is also why your test case fails.

Reply:
Okay, I'll apply the changes. I think the reason why I hadn't be able to see the travis logs was due to in some cases it take some time to be accessible.